orca-webhook/src/main/groovy/com/netflix/spinnaker/orca/webhook/tasks/CreateWebhookTask.groovy

```diff
@@ -138,8 +138,7 @@ class CreateWebhookTask implements RetryableTask {
             break
           case WebhookProperties.StatusUrlResolution.webhookResponse:
             try {
-              JsonPath path = JsonPath.compile(stageData.statusUrlJsonPath as String)
-              statusUrl = new JsonContext().parse(response.body).read(path)
+              statusUrl = JsonPath.compile(stageData.statusUrlJsonPath as String).read(response.body)
```

Comment:
That `JsonContext` constructor has changed and requires a bunch of parameters now. Just using the static `read` method appears to be the problem this code was introduced to avoid, where there's some problem with the `JsonPath` not being immutable.